HW_1/Task_1.py

```diff
@@ -0,0 +1,22 @@
+TEXT = f"Введите длинну стороны треугольника "
+NOT_EXIST = "Tреугольника с такими сторонами не существует!"
+a = int(input(TEXT + "№1: "))
+b = int(input(TEXT + "№2: "))
+c = int(input(TEXT + "№3: "))
+
+check1 = a + b
+check2 = a + c
+check3 = b + c
+
+if (a > 0 and b > 0 and c > 0):
```

Comment:
Скобки не обязательны
```suggestion
if a > 0 and b > 0 and c > 0:
```